tests/cluster-gpu-test-job.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cluster-gpu-tests
+spec:
+  parallelism: 4 # DYNAMIC_PARALLELISM
+  completions: 4 # DYNAMIC_COMPLETIONS
+  backoffLimit: 5
+  template:
+    spec:
+      containers:
+        - name: tensorflow-container
+          image: nvcr.io/nvidia/cuda:10.0-devel
```

Comment:
Is 10.0 a reasonable firmware version or should we go lower to increase compatability?